package.json

```diff
@@ -13,23 +13,23 @@
   },
   "dependencies": {
     "@babel/compat-data": "^7.19.1",
-    "@bcrs-shared-components/breadcrumb": "2.1.2",
-    "@bcrs-shared-components/business-lookup": "1.1.2",
-    "@bcrs-shared-components/certify": "2.1.2",
-    "@bcrs-shared-components/completing-party": "2.1.2",
+    "@bcrs-shared-components/breadcrumb": "2.1.4",
+    "@bcrs-shared-components/business-lookup": "1.1.4",
+    "@bcrs-shared-components/certify": "2.1.4",
+    "@bcrs-shared-components/completing-party": "2.1.4",
     "@bcrs-shared-components/confirm-dialog": "1.2.1",
-    "@bcrs-shared-components/contact-info": "1.2.2",
+    "@bcrs-shared-components/contact-info": "1.2.4",
     "@bcrs-shared-components/corp-type-module": "1.0.9",
-    "@bcrs-shared-components/court-order-poa": "2.1.2",
-    "@bcrs-shared-components/date-picker": "1.2.2",
+    "@bcrs-shared-components/court-order-poa": "2.1.4",
+    "@bcrs-shared-components/date-picker": "1.2.4",
     "@bcrs-shared-components/document-delivery": "1.1.1",
-    "@bcrs-shared-components/effective-date-time": "1.1.2",
-    "@bcrs-shared-components/enums": "1.0.24",
+    "@bcrs-shared-components/effective-date-time": "1.1.4",
+    "@bcrs-shared-components/enums": "1.0.26",
```

Comment:
I added enums and the other components are simply up-versioned. (They should not be up-versioned if they haven't changed -- though some do depend on shared enums -- but this is what happens when I accidentally do a "git reset" on my main branch in the shared repo.)